integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep24End2EndTests.kt

```diff
@@ -0,0 +1,85 @@
+package org.stellar.anchor.platform.test
+
+import io.ktor.client.*
+import io.ktor.client.plugins.*
+import io.ktor.client.request.*
+import junit.framework.TestCase.assertEquals
+import junit.framework.TestCase.fail
+import kotlin.time.Duration.Companion.seconds
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.runBlocking
+import org.stellar.anchor.platform.CLIENT_WALLET_SECRET
+import org.stellar.anchor.platform.TestConfig
+import org.stellar.anchor.util.Sep1Helper
+import org.stellar.walletsdk.ApplicationConfiguration
+import org.stellar.walletsdk.StellarConfiguration
+import org.stellar.walletsdk.Wallet
+import org.stellar.walletsdk.anchor.TransactionStatus
+import org.stellar.walletsdk.asset.IssuedAssetId
+import org.stellar.walletsdk.auth.AuthToken
+import org.stellar.walletsdk.horizon.SigningKeyPair
+
+class Sep24End2EndTest(
+  private val config: TestConfig,
+  private val toml: Sep1Helper.TomlContent,
+  private val jwt: String
+) {
+  private val walletSecretKey = System.getenv("WALLET_SECRET_KEY") ?: CLIENT_WALLET_SECRET
+  private val keypair = SigningKeyPair.fromSecret(walletSecretKey)
+  private val wallet =
+    Wallet(StellarConfiguration.Testnet, ApplicationConfiguration(useHttp = true))
+  private val USDC =
+    IssuedAssetId("USDC", "GDQOE23CFSUMSVQK4Y5JHPPYK73VYCNHZHA7ENKCV37P6SUEO6XQBKPP")
+  private val asset = USDC
+  private val client =
+    HttpClient() {
+      install(HttpTimeout) {
+        requestTimeoutMillis = 300000
+        connectTimeoutMillis = 300000
+        socketTimeoutMillis = 300000
+      }
+    }
+  private val anchor = wallet.anchor(config.env["anchor.domain"]!!.substring("http://".length))
```

Comment:
FYI:
In 0.6 you will be able to just use `wallet.anchor(config.env["anchor.domain"]!!`

Reply:
That would be lovely!